features/languages-testcase.feature

```diff
@@ -0,0 +1,39 @@
+         Feature: Languages
+
+Scenario Outline: User is able to change the language
+           Given User sees <country-flag> on the pah-fm website
+            When User clicks on <country-flag>
+            Then The language on the page changes to <chosen-language>
+
+        Examples:
+|    country-flag | chosen-language |
+|         Ukraine |       Ukrainian |
+|  United Kingdom |         English |
+|          Poland |          Polish |
+
+Scenario Outline: Language is retained after User login
```

Comment:
Is that business-logic related? Do we have ticket reference in there?

Reply:
Add logout #91 ?

Reply:
We can keep it for now.